### PR TITLE
Feature: Admissions viewbook

### PR DESCRIPTION
**Pull request by bspeare**
# How to test

- Verify that https://uids.brand.uiowa.edu/branches/feature_admissions_viewbook/components/detail/viewbook.html is mostly functional (cta icons are missing)
- Verify that `Aerial` and `Grid panel` components have been moved within a `Viewbook` directory
- Review changes on https://github.com/uiowa/uids/pull/292/files

### CODE REVIEW

**Comment by bspeare** [comment]
Next steps:

- [ ] Update class names used on https://localhost:3000/components/detail/viewbook
- [ ] Document changes made in layout that need to be made in viewbook
- [ ] move `Aerial` and `Grid panel` into a hidden viewbook folder

File: src/components/button/button.scss [hunk 1]
```diff
@@ -311,3 +311,5 @@
     color: inherit;
   }
 }
+
+
```
Comment (pyrello) [inline src/components/button/button.scss:315]: Remove extra line?

File: src/components/hero/README.md [hunk 2]
```diff
@@ -4,4 +4,5 @@ A hero consists of a large image or video with content sitting on top of it.
 **Must contain an image field and either:**
 * Title *and/or*
 * Body
+* Link
```
Comment (pyrello) [inline src/components/hero/README.md:8]: We could probably just remove this file?

**Review by pyrello — APPROVED** [review]
Reviewed, not tested.

**Pull request merged**